docs/flow/components/targets.mutate.md

```diff
@@ -0,0 +1,102 @@
+# targets.mutate
+
+The `targets.mutate` component rewrites the label set of the input
+targets by applying one or more `relabel_config` steps. If no relabeling
+steps are defined, then the input targets will be exported as-is.
+
+The most common use of `targets.mutate` is to filter Prometheus targets
+or standardize the label set that will be passed to a downstream component.
+The `relabel_config` blocks will be applied to the label set of each target in
+order of their appearance in the configuration file.
+
+Multiple `targets.mutate` components can be specified by giving them
+different name labels like "keep-backend-only" in the following example.
+
+## Example
+
+```hcl
+targets "mutate" "keep-backend-only" {
+  targets = [ 
+    { "__meta_foo" = "foo", "__address__" = "localhost", "instance" = "one",   "app" = "backend"  },
+    { "__meta_bar" = "bar", "__address__" = "localhost", "instance" = "two",   "app" = "database" },
+    { "__meta_baz" = "baz", "__address__" = "localhost", "instance" = "three", "app" = "frontend" }
+  ]
+  
+  relabel_config {
+    source_labels = ["__address__", "instance"]
+    separator     = "/"
+    target_label  = "destination"
+    action        = "replace"
+  } 
+  
+  relabel_config {
+    source_labels = ["app"]
+    action        = "keep"
+    regex         = "backend"
+  }
+}
+```
+
+## Arguments
+
+The following arguments are supported and can be referenced by other
+components:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+targets | list(map(string)) | The targets to mutate. | | no
+relabel_config | RelabelConfig | The relabeling steps to apply. | | no
```

Comment:
Hmm, for now, can we drop the block as being mentioned as an argument? I think blocks will have different documentation semantics than attributes.

```suggestion
```

Reply:
Yeah, I wasn't sure how to handle them. Makes sense.